### PR TITLE
solution

### PR DESCRIPTION
**Pull request by lukacsja**
[DEMO LINK](https://lukacsja.github.io/react_movies-list-filter/)

### CODE REVIEW

**Review by Rijo-J — APPROVED** [review]
Just optional suggestions ofc! Looks great! 😁 👍 🚀

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(
+    movie => movie.title.toLowerCase().includes(query.toLowerCase().trim())
+    || movie.description.toLowerCase().includes(query.toLowerCase().trim()),
+  );
```
[the remark this answers is not in the document]
Comment (lukacsja) [reply src/App.tsx:11]: 👍

File: src/App.tsx [hunk 2]
```diff
@@ -20,12 +26,13 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={(e) => setQuery(e.target.value)}
```
[the remark this answers is not in the document]
Comment (lukacsja) [reply src/App.tsx:29]: Good point It seems like i missed the checklist this time. 

Uploading the fixes thanks for the review

File: src/App.tsx [hunk 3]
```diff
@@ -20,12 +30,13 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={(changeEvent) => setQuery(changeEvent.target.value)}
```
Comment (naziamoff) [inline src/App.tsx:33]:
```suggestion
                onChange={(e) => setQuery(e.target.value)}
```

**Review by naziamoff — APPROVED** [review]
looks good 👍

**Pull request closed without merging**